libs/sdk-ui-dashboard/src/model/state/ui/index.ts

```diff
@@ -0,0 +1,18 @@
+// (C) 2021 GoodData Corporation
+import { createSlice } from "@reduxjs/toolkit";
+import { uiInitialState } from "./uiState";
+import { uiReducers } from "./uiReducers";
+
+const uiSlice = createSlice({
+    name: "uiSlice",
+    initialState: uiInitialState,
+    reducers: uiReducers,
+});
+
+export const uiReducer = uiSlice.reducer;
```

Comment:
small stuff: please rename to `uiSliceReducer`.. it is sort of a convention with the other slices

Reply:
Renamed